src/test/java/core/basesyntax/report/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.report;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.dataio.CsvToStorage;
+import core.basesyntax.dataio.FileReaderImpl;
+import core.basesyntax.dataio.FileWriter;
+import core.basesyntax.dataio.FileWriterImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.TreeMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportCreatorImplTest {
+    private static ReportCreator reportCreator = new ReportCreatorImpl();
+
+    @Before
+    public void setUp() {
+        Storage.setFruitsMap(new TreeMap<>());
+    }
+
+    @Test
+    public void test1_ReportCreatorOk() {
+        new CsvToStorage(new FileReaderImpl("src/main/resources/test1.cvs").read());
+        String actual = reportCreator.createReport();
```

Comment:
your services are hardly dependent on each other - let's test them separately
if you are testing `createReport()` method - don't read any files and use your own service to put data to storage

just put the data directly to the storage